app/ui/client/lib/fileUpload.js

```diff
@@ -260,7 +260,7 @@ export const fileUpload = async (files, input, { rid, tmid }) => {
 			confirmButtonText: t('Send'),
 			cancelButtonText: t('Cancel'),
 			html: true,
-			onRendered: () => $('#file-name').focus(),
+			onRendered: () => $('#file-description').focus(),
```

Comment:
I think this should not be changed, since the user might want to edit the name of the file before upload.

Reply:
The mentioned issue told to focus on description.